Gems/Meshlets/ASV_GPU_and_CPU_Demo/Readme.txt

```diff
@@ -116,14 +120,26 @@ Adding the meshoptimizer library to the Gem
 ===========================================
 The meshoptimizer library is not included as part of the O3DE.
 When adding and compiling this Gem, compile the meshoptimizer 
-library and add it as part of Meshlets.Static project (for example, in 
-Visual Studio you simply add the library file to the project).
+library and add it as part of Meshlets.Static project.
 Once this is done, the Gem should compile and link properly to allow you 
-to run the ASV sample 'Meshlets' (created with the MeshletsExampleComponent)
+to run the ASV sample 'Meshlets' (created with the MeshletsExampleComponent).
 
 The meshoptimizer library and source can be found in the following Github link:
 https://github.com/zeux/meshoptimizer
 
+Here are the specific steps to build and install the library, on Windows:
```

Comment:
Great add - this should be referenced at the previous section as indicated.

Reply:
That's what I meant by "This is a summary of the steps you'll need to follow, see the additional sections below for details."